main/app-menu.js

```diff
@@ -1,16 +1,43 @@
-const { Menu, MenuItem, ipcMain } = require('electron')
+const fs = require('node:fs/promises')
+const { Menu, MenuItem, ipcMain, dialog, BrowserWindow } = require('electron')
 const { ipcMainEvents } = require('./ipc')
+const { getLog } = require('./saturn-node')
 
 function setupAppMenu (/** @type {import('./typings').Context} */ ctx) {
-  // Add "Check for updates..." item to the Application menu on MacOS
+  const menu = Menu.getApplicationMenu()
+  if (!menu) return
+
+  setupCheckForUpdatesMenuItem(ctx, menu)
+
+  menu.items[1].submenu?.insert(1, new MenuItem({
+    label: 'Save Module Logs As…',
```

Comment:
> I guess I was thinking that in the future this would export all logs

Makes sense 👍🏻

In which case, I think `Save Saturn Module Log As...` (singular form - Log) would be even better.